pages/events.jsx

```diff
@@ -63,6 +128,23 @@ var EventsPage = React.createClass({
                 <p className="callout-heading">Check out the highlights from Maker Party and see more photos in our <a href="https://www.flickr.com/photos/mozilladrumbeat/galleries/72157643962655534/">Flickr gallery</a>.</p>
               </div>
             </div>
+            <PageEndCTA
+            header=""
+            dividerImgSrc="/img/pages/events/svg/line-divider.svg">
+              <div className="row" id="mailinglist">
+                { (this.context.router.getCurrentQuery().mailinglist === "thanks")
+                 ?
+                  <div className="thank-you-message">
```

Comment:
Since this class is only used for testing, could you call it maybe `test-thank-you-message' or something similar, to indicate that it's only used for testing, and not for CSS styling?

Alternatively, you could just do `node.textContent.should.match(/thank you/i)` of an ancestor node in your test suite.